libraries/AP_Camera/AP_Camera.h

```diff
@@ -88,6 +92,8 @@ class AP_Camera {
     AP_Int16        _servo_off_pwm;     // PWM value to move servo to when shutter is deactivated
     uint8_t         _trigger_counter;   // count of number of cycles shutter has been held open
     AP_Relay       *_apm_relay;         // pointer to relay object from the base class Relay.
+    AP_Int8         _auto_only;         // if 1: trigger by distance only if in AUTO mode.
+    bool            _is_auto;           // true if in AUTO mode
```

Comment:
Nitpicking: changing this to:

- `_auto_mode_only`
- `_is_in_auto_mode`

looks better to me.